examples/auto_instrumentation/README.md

```diff
@@ -0,0 +1,76 @@
+# Overview
+
+This example shows how to use the auto-instrumentation agent in OpenTelemetry.
+
+A uninstrumented script will be executed once without the agent and then a instrumented script will
+be run with the agent. The results should show a `Span` being started in both cases.
```

Comment:
An uninstrumented script without the agent and an instrumented script with the agent showing spans seems confusing, maybe the text needs clarifying here?

Reply:
Yeah... will fix that.

Reply:
Fixed, I wrote again a big part of the example, hopefully it is more clear now.